### DISCUSSION TITLE
opsgenie_addr cannot be formatted with fields from the first match

### DISCUSSION THREAD

**Discussion by mohamedelema17** [jertel/elastalert2#1561] [Q&A]
It appears that opsgenie_addr cannot be formatted with fields from the first match.

Steps to reproduce:-
- Set `opsgenie_addr: "{_id}"`

You will get   
`ERROR:elastalert:Error while running alert opsgenie: Error sending alert: Invalid URL '{_id}': No scheme supplied. Perhaps you meant https://{_id}?`

Expectation:
```
ERROR:elastalert:Error while running alert opsgenie: Error sending alert: Invalid URL '9409d72a-5954-4d4f-9f3b-f77de0530298
': No scheme supplied. Perhaps you meant https://9409d72a-5954-4d4f-9f3b-f77de0530298?
```

NB: the value of _id is 9409d72a-5954-4d4f-9f3b-f77de0530298 which is also my opsgenie_alias. I want to dynamically generate opsgenie_addr to something like "https://api.opsgenie.com/v2/alerts/{_id}/close?identifierType=alias". However, if I do this, {_id} is not replaced.

**Comment by jertel** [comment]
The author of the opsgenie alerter chose to not support dynamic values via field substitution for the `opsgenie_addr` field. You're welcome to submit a PR to add this feature. If you would like to pursue that please follow the contribution guidelines located at the root of this project.

**Comment by mohamedelema17** [comment]
@jertel here is the PR https://github.com/jertel/elastalert2/pull/1563
